src/editor/tilebox.cpp

```diff
@@ -108,7 +109,7 @@ EditorTilebox::draw_tilegroup(DrawingContext& context)
     auto position = get_tile_coords(pos, false);
     m_editor.get_tileset()->get(tile_ID).draw(context.color(), position, LAYER_GUI - 9);
 
-    if (g_config->developer_mode && m_active_tilegroup->developers_group)
+    if (g_config->developer_mode && (m_active_tilegroup->developers_group || g_debug.show_toolbox_tile_ids) && tile_ID != 0)
```

Comment:
Showing 0 in empty tiles is useless information

Reply:
Technically speaking 0 was never shown before because 0 can't appear in the `others` group. The reason I added this is because it looks neater in other groups that do have 0s.

> Showing 0 in empty tiles is useless information

Removing 0 also makes it easier to spot empty tiles that aren't 0.